tool/src/lib.rs

```diff
@@ -82,6 +84,21 @@ pub fn gen(
 
     match target_language {
         "js" => js::gen_bindings(&env, &mut out_texts, Some(docs_url_gen)).unwrap(),
+        "kotlin" => {
+            let mut attr_validator = hir::BasicAttributeValidator::new("kotlin");
+            attr_validator.support.renaming = true;
+            attr_validator.support.disabling = true;
+            let tcx = match hir::TypeContext::from_ast(&env, attr_validator) {
+                Ok(context) => context,
+                Err(e) => {
+                    for err in e {
+                        eprintln!("Lowering error: {}", err);
+                    }
+                    std::process::exit(1);
+                }
+            };
+            out_texts = kotlin::run(&tcx, "dev.gigapixel", "somelib").take_files();
```

Comment:
nit: probably best to have this be taken from the CLI, or a config, in the long run. fine for now

we need to come up with a principled model for backend-specific config. Right now we have this `library_config` that you can set which is fine to use.

Reply:
agreed. Didn't really know how to pick those up. I'd like it if the actual library name could be picked up from Cargo.toml (at least as a default), but couldn't figure that out yet.